### ISSUE TITLE
How to deal with unsorted insect specimens

### ISSUE THREAD

**Issue by mbprondzinski** [ArctosDB/arctos#2056]
Another question for you:  If I have an accession that includes 10 vials of unsorted insects in lots, do I have to assign a catalog number to each of those 10 vials?  Can I retain them simply as "lot of 20 insects; lot of 32 insects; lot of 'many insects'" with just the accession number as an identifier, or even a collector #, and only assign the catalog numbers after they've been sorted?

Thanks.
MB

**Comment by dustymc** [comment]
>do I have to assign a catalog number to each of those 10 vials?

No, cataloged items - the things that get catalog numbers - are explicitly "something someone felt like cataloging for some reason."

>assign the catalog numbers after they've been sorted

With this included, I think it's a completely defensible approach. "5 gallons of things that showed up at the light trap" should never be cited, but it can certainly be loaned and you can work with the loan-ee to get good identifiers for the individuals they want to use/cite.

http://arctos.database.museum/info/ctDocumentation.cfm?table=CTID_REFERENCES&field=same%20lot%20as exists to keep that all linked together.

**Comment by Jegelewicz** [comment]
> If I have an accession that includes 10 vials of unsorted insects in lots, do I have to assign a catalog number to each of those 10 vials?

That depends. Did they all come from the exact same locality at the same time? Were they collected by the same person? Do you have any kind of taxonomic identification for any of them?

If they are from a single collecting event, then I would assign one catalog number to all 10 vials (is there one insect in each vial?) and use the part name "whole organism" with whatever preservation fluid they are in or dry if that's how they are preserved and lot count of "10". If they are not identified and the best you can do is "Insecta", that is fine for an identification.

> Can I retain them simply as "lot of 20 insects; lot of 32 insects; lot of 'many insects'" with just the accession number as an identifier, or even a collector #, and only assign the catalog numbers after they've been sorted?

Add any other identifiers that you have to the catalog record as they generally come in handy in the future. You don't need to add an accession number as the catalog number should be part of whatever accession you entered them with.

If at any time someone comes along and identifies one of the insects in this lot, give that individual specimen it's own catalog number and then create the relationship "same lot as" to the original catalog number.

If this is all clear as mud, let's talk on the phone!

OR

Send me an example and I'll enter it for you so you can see what I mean.

**Comment by campmlc** [comment]
We do this with parasites that have not been sorted and IDd yet. It would
be one catalog number, ID Phylum Arthropoda for example, whole organism
part, lot count of xxx.

**Comment by mbprondzinski** [comment]
Now that I'm finally having the time to read the Arctos handbook, it is starting to make sense.  We have a collection of insects that were donated by one of our curators (to which one accession number was assigned the day it was received) and he was wondering how to deal with the unsorted stuff in Arctos.  I wasn't sure if he needed to assign a catalog number to the unsorted stuff or simply let it lie until it could be identified.  But assigning all the unsorted vials to one catalog number initially and then assigning individual catalog numbers to the specimens once they are identified would work.
